[IMP] helpdesk: update help center documentation (#3864)
odoo/documentation: content/applications/services/helpdesk/overview/help_center.rst

   *Installing a new application on a One-App-Free database will trigger a 15-day trial. At the end
   of the trial, if a paid subscription has not been added to the database, it will no longer be
   accessible.*

Knowledge
=========

Odoo's :guilabel:`Knowledge` application is a collaborative library where users can store, edit, and
share information. The :guilabel:`Knowledge` app is represented throughout the database by the
*book* icon.

.. image:: help_center/help-center-knowledge-book-icon.png
   :align: center
   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.

Enable knowledge on a helpdesk team
-----------------------------------

First, go to :menuselection:`Helpdesk --> Configuration --> Teams` and select a team, or create a
:doc:`new one <getting_started>`. On the team's settings page, scroll to :guilabel:`Self-Service`
and enable :guilabel:`Knowledge`. If prompted, manually save and refresh the page to activate the
feature. Once the page has refreshed, a field labeled :guilabel:`Article` will appear.

Odoo automatically creates an article titled `Help` when this feature is activated. Select it from

> the drop-down to enable it.

Author: Thank you for catching this! I spoke with the PO and did some additional testing and it appears there are visibility constraints for these features, so I added a note block at the beginning of the doc.

   (:guilabel:`+`) next to :guilabel:`Workspace`.

Once an article has been created and assigned to a :guilabel:`Helpdesk team`, content can be
:doc:`added <../../../productivity/knowledge/articles_editing>` and organized through the
:guilabel:`Knowledge` app.

Searching articles from a helpdesk ticket
-----------------------------------------

When members of a :guilabel:`Helpdesk team` are trying to solve a ticket, they can search through
the content in the :guilabel:`Knowledge` app for more information on the issue they are facing.

First, open a :guilabel:`ticket`, either from the :guilabel:`Helpdesk` dashboard or by going to
:menuselection:`Helpdesk --> Tickets --> All Tickets` and selecting a :guilabel:`ticket` from the
list. Click on the :guilabel:`Knowledge` icon above the :guilabel:`chatter` for the
:guilabel:`ticket`. This will open the search window.

.. image:: help_center/help-center-knowledge-search.png
   :align: center
   :alt: View of knowledge search window from a helpdesk ticket.

.. tip::
   :guilabel:`Knowledge` articles can also be searched by clicking `Ctrl + K` to open

>    the command pallette, then `?` to narrow the search to :guilabel:`articles`.

Author: switching this up to :command: since these are user inputs

the feature. Once the page has refreshed, a field labeled :guilabel:`Forums` will appear.

Odoo automatically creates a forum titled `Help` when this feature is activated. Select it from the
drop-down to enable it. To create a new forum, or add an additional forum, type the name into the
field, and click :guilabel:`Create and edit`.

Setting up a forum
------------------

The settings on a :guilabel:`Forum` are managed through either the :guilabel:`Website` or
:guilabel:`eLearning` applications. From the Odoo dashboard, navigate to :menuselection:`Website -->
Configuration --> Forums` or :menuselection --> eLearning --> Forum --> Forums`. In either case,
select a forum to edit from the list.

Select a mode. :guilabel:`Questions` mode allows for one answer/response for each post, while
:guilabel:`Discussion` mode allows for multiple responses. To limit the forum to a single website,
select it from the :guilabel:`Website`.

Under the :guilabel:`Order and Visibility` section,choose how the posts should be sorted by updating
the :guilabel:`Default Sort` field. Change the :guilabel:`Privacy` setting to change what users will

> have access to the forum.

Author: The visibility field/setting only appears for forums that are tied directly to eLearning courses, not to `Help` forums. So if they are following the steps to set up or update a help forum, they shouldn't see those fields. (I do not know if it is necessary to include a note about that as it veers too much into "eLearning" documentation and not help center specific)

the course in sections.

.. image:: help_center/help-center-elearning-course-contents-page.png
   :align: center
   :alt: View of a course being published for Odoo Helpdesk.

.. note::
   In order to add a certification to a course, go to :menuselection:`eLearning --> Configuration
   --> Settings`, check the box labeled :guilabel:`Certifications`, and :guilabel:`Save` to activate
   the setting.

.. seealso::
   `Odoo Tutorials: eLearning <https://www.odoo.com/slides/elearning-56>`_

Publish an eLearning course
---------------------------

To allow customers to enroll in a course, both the course and the contents need to be published.

If the course is published, but the contents of the course are not published, customers can enroll
in the course on the website, but they won't be able to view any of the course content. Knowing
this, it may be beneficial to publish the course first if the course contents are intended to be
released over time, such as classes with a weekly schedule.

To make the entire course available at once, each piece of course content must be published first,
then the course can be published.

> To publish an course, choose a course from the *eLearning* dashboard. On the course template page,

Reviewer: typo 
```suggestion
To publish a course, choose a course from the *eLearning* dashboard. On the course template page,
```

> ===========
> Help Center
> ===========
> 
> Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *Help
> Center*.

Reviewer 2: Since "help center" is not an app/module, an element on the UI, or a proper noun and is just an umbrella term for the combined resources and features between Helpdesk, Knowledge, Forums, and eLearning I believe it should be sentence case throughout the document :)
```suggestion
===========
Help center
===========

Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *help
center*.
```

   of the trial, if a paid subscription has not been added to the database, it will no longer be
   active or accessible.

Knowledge
=========

Odoo's *Knowledge* application is a collaborative library where users can store, edit, and share
information. The *Knowledge* app is represented throughout the database by a *book* icon.

.. image:: help_center/help-center-knowledge-book-icon.png
   :align: center
   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.

Enable knowledge on a Helpdesk team
-----------------------------------

To enable the *Knowledge* feature on a *Helpdesk* team, go to :menuselection:`Helpdesk -->
Configuration --> Teams` and select a team, or create a :doc:`new one <getting_started>`.

When a team has been selected (or created), Odoo displays that team's detail form.

On the team's detail form, scroll down to the :guilabel:`Self-Service` section. Click the box next
to :guilabel:`Knowledge` to activate the *Knowledge* feature. When clicked, a new field labeled
:guilabel:`Article` appears.

Clicking the :guilabel:`Article` field reveals a drop-down menu. At first, there is only one option

> in the drop-down menu titled *Help*, which Odoo provides by default. Select *Help* from the

Reviewer 2: This is a element on the UI with active instruction - please use guilabels 
```suggestion
in the drop-down menu titled :guilabel:`Help`, which Odoo provides by default. Select :guilabel:`Help` from the
```

drop-down to choose this article.

.. tip::
   To create a new article, go to :menuselection:`Main Dashboard --> Knowledge`. In the *Knowledge*
   app, hover the cursor next to the :guilabel:`Workspace` section header, located in the left
   sidebar. Moving the cursor there reveals a hidden *plus sign* icon (:guilabel:`+`).

   Click the "plus sign" icon (:guilabel:`+`) to create a new article in the :guilabel:`Workspace`.
   In the upper right corner of the page, click the :guilabel:`Share` button, and slide the
   :guilabel:`Share to Web` toggle switch until it reads :guilabel:`Article Published`. It can then
   be added to a *Helpdesk* team.

Once an article has been created and assigned to a *Helpdesk* team, content can be added and
organized through the *Knowledge* app.

.. seealso::
   - :doc:`Editing Knowledge articles </applications/productivity/knowledge/articles_editing>`

Search articles from a Helpdesk ticket
--------------------------------------

When members of a *Helpdesk* team are trying to solve a ticket, they can search through the content
in the *Knowledge* app for more information on the issue.

> To search *Knowledge* articles, open a :guilabel:`ticket` - either from the *Helpdesk* app
> dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All Tickets`, and selecting a
> :guilabel:`Ticket` from the list.

Reviewer: no gui on line 79
em (—) dash instead of hyphen

```suggestion
To search *Knowledge* articles, open a ticket — either from the *Helpdesk* app
dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All Tickets`, and selecting a
:guilabel:`Ticket` from the list.
```

When a :guilabel:`Ticket` is selected, Odoo reveals that ticket's detail form.

> Click the :guilabel:`Knowledge` *book* icon, located above the chatter to open a search window.

Reviewer: @jero-odoo none of these worked?
https://emojipedia.org/search/?q=book

Otherwise what you have *seems* ok at a quick glance...

to the ticket's description.

To use templates in a *Helpdesk* ticket, first, open a ticket, either from the :guilabel:`Helpdesk`
dashboard or by going to :menuselection:`Helpdesk --> Tickets --> All Tickets` and selecting a
ticket from the list.

Click on the :guilabel:`Knowledge` *book* icon above the chatter for the ticket. This opens a search
window. In this search window, select (or search for) the desired article. Doing so reveals that
article page in the Odoo *Knowledge* application.

To use a template to respond to a ticket, click :guilabel:`Send as message` in the upper right
corner of the template box, located in the body of the article.

Doing so opens a :guilabel:`Compose email` pop-up window. In ths window, select the recipients, make
any necessary additions (or edits) to the template, then click :guilabel:`Send`.

.. tip::
   To use a template to add information to a ticket's description, click :guilabel:`Use as
   description` in the upper right corner of the template box, located in the body of the article.
   Doing so will not replace the existing text in a ticket's description. The template will be added
   as additional text.

Community Forum
===============

> A *Community Forum*` provides a space for customers to answer each other's questions and share

Reviewer 2: extra backtick here

   :align: center
   :alt: Overview of the settings page of a team emphasizing the help center features.

The *help center* is a centralized location where teams and customers can search for (and share)
detailed information about products and services.

.. note::
   In order to activate any of these features on a *Helpdesk* team, (*Forums*, *eLearning*, or
   *Knowledge*), the :guilabel:`Visibility` of the team has to be set to :guilabel:`Invited portal
   users and all internal users`. See :doc:`Getting Started
   </applications/services/helpdesk/overview/getting_started>` for more information on *Helpdesk*
   team settings and configuration.

.. warning::
   Since all of the *help center* features require integration with other applications, enabling any
   of them may result in the installation of additional modules (or applications).

   Installing a new application on a *One-App-Free* database will trigger a 15-day trial. At the end
   of the trial, if a paid subscription has not been added to the database, it will no longer be
   active or accessible.

Knowledge
=========

Odoo's *Knowledge* application is a collaborative library where users can store, edit, and share

> information. The *Knowledge* app is represented throughout the database by a *book* icon.

Reviewer: you could probably get away with gray notebook emoji here since it's so close 👀 
up to you tho
```suggestion
information. The *Knowledge* app is represented throughout the database by a *📓 (book)* icon.
```

Author: it _is_ close, but I think I am going to leave it out, just to avoid any confusion.

.. image:: help_center/help-center-knowledge-book-icon.png
   :align: center
   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.

> Enable knowledge on a Helpdesk team

Reviewer: wondering if we do uppercase here, since what we're saying here is "Integrate the Knowledge App with a team from the Helpdesk app"

wdyt?

```suggestion
Enable Knowledge on a Helpdesk team
```

-----------------------------------

To enable the *Knowledge* feature on a *Helpdesk* team, go to :menuselection:`Helpdesk -->
Configuration --> Teams` and select a team, or create a :doc:`new one
</applications/services/helpdesk/overview/getting_started>`.

When a team has been selected (or created), Odoo displays that team's detail form.

On the team's detail form, scroll down to the :guilabel:`Self-Service` section. Click the box next
to :guilabel:`Knowledge` to activate the *Knowledge* feature. When clicked, a new field labeled
:guilabel:`Article` appears.

Clicking the :guilabel:`Article` field reveals a drop-down menu. At first, there is only one option
in the drop-down menu titled :guilabel:`Help`, which Odoo provides by default. Select
:guilabel:`Help` from the drop-down to choose this article.

.. tip::

>    To create a new article, go to :menuselection:`Main Dashboard --> Knowledge`. In the *Knowledge*
>    app, hover the cursor next to the :guilabel:`Workspace` section header, located in the left
>    sidebar. Moving the cursor there reveals a hidden :guilabel:`+ (plus sign)` icon.

Reviewer: Is there a use case you've seen where **menuselection** instructions** include "Main Dashboard?
I ask because I've never seen it before, and the **menuselection** case followed by *italics* when we're actively looking at that dashboard looks strange imo because the formatting sends mixed messages.

To be clear, this paragraph is *technically* correct.

However here are my suggestion for improvement around that **menuselection** tag, along with a word change (header >> heading), and using an emoji.

```suggestion
   To create a new article, go to the :menuselection:`Knowledge app`, then hover the cursor next to
   the :guilabel:`Workspace` section heading, located in the left sidebar. Moving the cursor there
   reveals a hidden :guilabel:`➕ (plus sign)` icon.
```

Author: I think this was the result of one too many suggested edits, I will adjust.

Click the :guilabel:`Knowledge (book)` icon, located above the chatter to open a search window.

.. image:: help_center/help-center-knowledge-search.png
   :align: center
   :alt: View of knowledge search window from a helpdesk ticket.

.. tip::
   *Knowledge* articles can also be searched by pressing :command:`Ctrl + K` to open the command
   palette, then typing :command:`?`, followed by the name of the desired article.

When Odoo reveals the desired article, click it, or highlight the :guilabel:`Article` title, and
press :command:`Enter`. This will open the article in the :guilabel:`Knowledge` application.

To open the article in a new tab, press :command:`Ctrl + Enter`.

.. tip::
   If a more in-depth search is required, press :command:`Alt + B`. That will reveal a separate
   page, in which a more detailed search can occur.

Share articles to the help center
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In order for a *Knowledge* article to be available to customers and website visitors, it has to be
published.

.. note::
   Even though the *Help* article has been enabled on a team, Odoo will not share all the nested

>    articles to the web. Individual articles intended for customers *must* be published for them to

Reviewer: nitpick
we're using italics mainly for product or feature names
bold, then, would be better for emphasis 
```suggestion
   articles to the web. Individual articles intended for customers **must** be published for them to
```

   be viewable on the website.

To publish an article, navigate to the desired article, (by following the above steps), and click

> :guilabel:`Share` icon in the upper-right corner. This will reveal a menu. Slide the toggle button

Reviewer: missing an article here

click...
_the_ Share icon...

labeled :guilabel:`Share to Web` to read :guilabel:`Article Published`.

.. image:: help_center/help-center-knowledge-sharing.png
   :align: center
   :alt: View of a knowledge article focused on sharing and publishing options.

Solve tickets with templates
----------------------------

> :guilabel:`Template` boxes can be added to *Knowledge* articles to allow content to be reused,

Reviewer: why gui here?

double-check all of the guis in this **Add templates to articles** section...are they needed or justified?

Specifically, are there fields or labels named **Template** or **Templates**?

In the following section, the words "template" and "template" are not capitalized and don't have guis...

```suggestion
*Template* boxes can be added to *Knowledge* articles to allow content to be reused,
```

--> Help`). Click on an existing nested article or create a new one by clicking the :guilabel:`+
(plus sign)` icon next to *Help*.

Type `/` to open the :guilabel:`Powerbox` and view a list of :doc:`commands
</applications/productivity/knowledge/articles_editing>`. Select or type `template`. A gray template
block will be added to the page. Add any necessary content to this block.

.. image:: help_center/help-center-knowledge-template-options.png
   :align: center
   :alt: View of a template in knowledge with focus on send and copy options.

.. note::
   :guilabel:`Templates` will only display the :guilabel:`Use as description` or :guilabel:`Send as
   Message` options if they are accessed from the :guilabel:`Helpdesk`.

Use templates in tickets
~~~~~~~~~~~~~~~~~~~~~~~~

Templates can be used to respond directly to a *Helpdesk* ticket as a message, or to add information
to the ticket's description.

To use templates in a *Helpdesk* ticket, first, open a ticket, either from the :guilabel:`Helpdesk`
dashboard or by going to :menuselection:`Helpdesk --> Tickets --> All Tickets` and selecting a

> ticket from the list.

Reviewer: How come in this section, _template_ and _templates_ are lowercase, with no guis?
How come the word _ticket_ is not capitalized with a gui like it was in previous sections?

Click on the :guilabel:`Knowledge (book)` icon above the chatter for the ticket. This opens a search

> window. In this search window, select (or search for) the desired article. Doing so reveals that

Reviewer: again with the odd parenthesis use
when we do things like this that are not normal it brings unnecessary attention to them
```suggestion
window. In this search window, select, or search for, the desired article. Doing so reveals that
```

      complete, without having a specific level of karma points. Those activities are:

      - Ask questions
      - Answer questions
      - Upvote
      - Downvote
      - Edit own posts
      - Edit all posts
      - Close own posts
      - Close all posts
      - Delete own posts
      - Delete all posts
      - Nofollow links
      - Accept an answer on own question
      - Accept an answer to all questions
      - Editor features: image and links
      - Comment on own posts
      - Comment on all posts
      - Convert own answers to comments (and vice versa)
      - Convert all answers to comments (and vice versa)
      - Unlink own comments
      - Unlink all comments
      - Ask questions without validation
      - Flag a post as offensive
      - Moderate posts
      - Change question tags
      - Create new tags

Create a forum post from a Helpdesk ticket
------------------------------------------

When a *Helpdesk* team has a *Forum* enabled, tickets submitted to that team can be converted to
forum posts.

To do that, select a ticket (either from a team's pipeline or from :menuselection:`Tickets --> All

> Tickets` in the *Helpdesk* application).

Reviewer 2: ```suggestion
Tickets` in the :guilabel:`Helpdesk` application).
```

> Tickets` in the :guilabel:`Helpdesk` application).

Reviewer 2: I apologize @jero-odoo, I meant a menuselection here not a guilabel in my previous correction.
```suggestion
Tickets` in the :menuselection:`Helpdesk` application).
```

To make the entire course available at once, each piece of course content must be published first,
then the course can be published.

To publish an *eLearning* course, navigate to a course's web page (:menuselection:`Dashboard -->
eLearning --> Select a Course --> Go to Website Smart Button` or :menuselection:`Dashboard -->
eLearning --> View Course`).

This will reveal the course's web page (on the front end). At the top of the course web page, move
the :guilabel:`Unpublished` toggle switch to :guilabel:`Published`.

Publish eLearning course contents from the back-end
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To publish *eLearning* course content from the back-end, go to the desired course's template page
(:menuselection:`eLearning--> Select the desired course`). On the course template page, click the
:guilabel:`Published Contents` smart button.

Doing so reveals a separate page displaying all the published content related to that course. Remove
the default :guilabel:`Published` filter from the search bar in the upper-right corner, to reveal
all the content related to the course - even the non-published content.

> Click the :guilabel:`List View (≣)` icon in the upper-right corner, directly beneath the search bar

Reviewer 2: Icon formatting
```suggestion
Click the :guilabel:`≣ (List View)` icon in the upper-right corner, directly beneath the search bar
```

:guilabel:`Helpdesk` app dashboard, or by going to :menuselection:`Helpdesk app --> Tickets --> All
Tickets`, and selecting a :guilabel:`Ticket` from the list.

When a :guilabel:`Ticket` is selected, Odoo reveals that :guilabel:`Ticket's` detail form.

Scroll down to the bottom of the detail form, and click the :guilabel:`Knowledge` *book* icon,
located above the :guilabel:`Chatter`. Clicking the :guilabel:`Knowledge` *book* icon opens a
search window.

.. image:: help_center/help-center-knowledge-search.png
   :align: center
   :alt: View of knowledge search window from a helpdesk ticket.

.. tip::
   :guilabel:`Knowledge` articles can also be searched by pressing :command:`Ctrl + K` to open
   the command pallette, then typing :command:`?`, followed by the name of the desired article.

When Odoo reveals the desired article, click it, or highlight the :guilabel:`Article` title, and
press :command:`Enter`. This will open the :guilabel:`Article` in the :guilabel:`Knowledge`
application.

To open the :guilabel:`Article` in a new tab, press :command:`Ctrl + Enter`.

.. tip::
   If a more in-depth search is required, press :command:`Ctrl + B`. That will reveal a separate

>    page, in which a more detailed search can occur.

Author: Turns out it is alt+b 👍

Share articles to the Help Center
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

In order for a :guilabel:`Knowledge` article to be available to customers and website visitors, it
has to be published.

.. note::

>    Even though the `Help` article has been enabled on a team, Odoo will not share all the nested

Author: I had originally been looking at this as a "user input", for which it would be correct to use backticks (as per @samueljlieber), however, I do agree it makes more sense to be italicized in this case.

Solve tickets with templates
----------------------------

:guilabel:`Template` boxes can be added to :guilabel:`Knowledge` articles and allow content to be
reused, copied, sent as messages, or added to the description on a ticket. This allows teams to
maintain consistency when answering customer tickets, and minimize the amount of time spent on
responding to repeat questions.

Add templates to articles
~~~~~~~~~~~~~~~~~~~~~~~~~

To create a :guilabel:`Template`, go to the `Help` article (:menuselection:`Dashboard --> Knowledge
--> Help`). Click on an existing nested article or create a new one by clicking the plus sign
(:guilabel:`+`) next to `Help`.

Type `/` to view a list of :doc:`commands </applications/productivity/knowledge/articles_editing>`.
Select or type `template`. A gray template block will be added to the page. Add any necessary
content to this block.

.. image:: help_center/help-center-knowledge-template-options.png
   :align: center
   :alt: View of a template in knowledge with focus on send and copy options.

.. note::
   :guilabel:`Templates` will only display the :guilabel:`Use as description` or :guilabel:`Send as

>    Message` options if they are accessed from the :guilabel:`Helpdesk`.

Author: I do not see this option (even in runbot on Master, so I am not sure how that came up?)

>    Message` options if they are accessed from the :guilabel:`Helpdesk`.

Reviewer 3: It depends on the article I believe, try looking at different knowledge articles

>    Message` options if they are accessed from the :guilabel:`Helpdesk`.

Author: It looks like that only applies to media, files or other imports, not specifically templates.

choose the course :guilabel:`Type` and :guilabel:`Visibility`. The :guilabel:`Visibility` setting
determines whether the course will be available to public site visitors or members.

Add content to an eLearning course
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To add content to a course, click the :guilabel:`Content` tab and select :guilabel:`Add Content`.
Choose the :guilabel:`Content Type` from the drop-down and upload the file (or past the link) where
instructed. Click :guilabel:`Save` when finished. Click :guilabel:`Add Section` to organize the
course in sections.

.. image:: help_center/help-center-elearning-course-contents-page.png
   :align: center
   :alt: View of a course being published for Odoo Helpdesk.

.. note::
   In order to add a certification to a course, go to :menuselection:`eLearning --> Configuration
   --> Settings` and check the box labeled :guilabel:`Certifications`, to activate the setting.

.. seealso::
   `Odoo Tutorials: eLearning <https://www.odoo.com/slides/elearning-56>`_

Publish an eLearning course
---------------------------

To allow customers to enroll in a course, both the course and the contents need to be published.

> If the course is published, but the contents of the course are not published, customers can enroll
> in the course, but they won't be able to view an of the course content. Knowing this, it may be
> beneficial to publish the course first if the course contents are intended to be released over time
> (such as classes with a weekly schedule).

Reviewer 3: ```suggestion
If the course is published, but the contents of the course are not published, customers can enroll
in the course, but they won't be able to view an of the course content. Knowing this, it may be
beneficial to publish the course first if the course contents are intended to be released over time
(such as classes with a weekly schedule).
```

Maybe specify where they enroll for the course

To make the entire course available at once, each piece of course content must be published first,
then the course can be published.

To publish an :menuselection:`eLearning` :guilabel:`course`, navigate to a course's web page
(:menuselection:`Dashboard --> eLearning --> Select a Course --> Go to Website Smart Button` or
:menuselection:`Dashboard --> eLearning --> View Course`).

This will reveal the course's web-page (on the front-end). At the top of the course web-page, move
the :guilabel:`Unpublished` toggle switch to :guilabel:`Published`.

Publish eLearning course contents from the back-end
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To publish :guilabel:`eLearning` course content from the back-end, go to the desired course's
template page (:menuselection:`Odoo Dashboard --> eLearning app --> Select the desired course`). On
the course template page, click the :guilabel:`Published Contents` smart button.

> Doing so reveals a separate page displaying all the published content related to that course.
> Remove the default :guilabel:`Published`` filter from the search bar in the upper-right corner, to
> all the content related to the course - even the non-published content.

Reviewer 3: ```suggestion
Doing so reveals a separate page displaying all the published content related to that course.
Remove the default :guilabel:`Published` filter from the search bar in the upper-right corner, to reveal
all the content related to the course - even the non-published content.
```

> Switch to :guilabel:`List View`, by clicking the "horizontal lines" icon in the upper right-corner,
> directly beneath the search bar.

Reviewer 3: ```suggestion
Switch to :guilabel:`List View`, by clicking the "hamburger menu (3 horizontal lines)" icon in the upper right-corner,
directly beneath the search bar.
```

While in :guilabel:`List View`, there is a checkbox on the far-left of the screen, above the listed
courses (to the left of the :guilabel:`Title` column). When that checkbox is clicked, all the
course contents are selected at once.

> With all the course content selected, click any of the boxes in the :guilabel:`Is Published`
> column. This reveals a pop-up window, asking for the confirmation that all selected records are
> intended to be published. Click :guilabel:`OK` to automatically publish all course content.

Reviewer 3: ```suggestion
With all the course content selected, double click any of the boxes in the :guilabel:`Is Published`
column. This reveals a pop-up window, asking for the confirmation that all selected records are
intended to be published. Click :guilabel:`OK` to automatically publish all course content.
```
